server/app/assets/javascripts/admin_question_edit.ts

```diff
@@ -8,6 +8,78 @@ class AdminQuestionEdit {
     if (modal !== null) {
       this.addCustomModalClickHandler(modal)
     }
+
+    const primaryApplicantInfoSection = document.getElementById(
+      'primary-applicant-info',
+    )
+    if (primaryApplicantInfoSection !== null) {
+      this.addUniversalToggleHandler(primaryApplicantInfoSection)
+      this.addEnumeratorDropdownHandler(primaryApplicantInfoSection)
+    }
+  }
+
+  addEnumeratorDropdownHandler(primaryApplicantInfoSection: HTMLElement) {
+    addEventListenerToElements(
+      '#question-enumerator-select',
+      'input',
+      (event: Event) => {
+        const target = event.target as HTMLInputElement
+        const setHidden = target.value !== ''
+        primaryApplicantInfoSection.toggleAttribute('hidden', setHidden)
+      },
+    )
+  }
+
+  addUniversalToggleHandler(primaryApplicantInfoSection: HTMLElement) {
```

Comment:
*Question (blocking)*: Do screenreaders do the right thing when we unhide elements or do we need to wrap all this in an `aria-live`?